### ISSUE TITLE
Broken recipes reported by OC (world sensor card)

### ISSUE THREAD

**Issue by llloyd4** [MightyPirates/OpenComputers#2446]
First off, let me say, this mod was added by the team who put together Direwolf20's mod pack.I really don't use it, I am not a programmer. But I don't know how far reaching this is so I am posting this mostly to see. So, when I load up my world, as of late, I have been getting the following in the chat log.

[01:11:08] [Client thread/INFO]: [CHAT] �aOpenComputers�f: There were errors loading one or more recipes. Some items may be uncraftable. Please check your log file for more information.

Not sure why or where, no crashes, but, I don't use OC so. The only crash I have now is with the latest version of Galacticraft, but I downgraded from Latest to Promoted and it *seems* to be stable, but that error above still pops up.

Minecraft: 1.10.2
Forge: 1.10.2-12.18.3.2316
OpenComputers: 1.10.2-1.6.2.7 

Console Log (WARNING!!! Long paste!!! ) - https://pastebin.com/Mmm4kyZt - Latest is on line 19608

**Comment by payonel** [comment]
`Failed adding recipe for 'worldSensorCard', you will not be able to craft this item`

**Comment by payonel** [comment]
this is fixed for 1.11 and 1.12, I tested the same fix in 1.10 and it didn't work. I scanned their mod for an item named sensor_lens or variations of that, nothing turned up. SO...i think I found its item name was, perhaps mistakenly, moved to `galacticraftcore:sensor_lens` in 1.10, and only 1.10

I'll make this change for 1.10, and leave it at that. Note that we don't plan to add support for GC in 1.10
